foundation/foundation-mda/src/main/resources/templates/deployment/spark-infrastructure/v2/spark.infrastructure.values.dev.yaml.vm

```diff
@@ -6,3 +6,12 @@ aissemble-spark-history-chart:
 
     ingress:
       enabled: false
+
+aissemble-hive-metastore-service-chart:
```

Comment:
I: We don't want to persist between tests for local dev

Reply:
Q: What do you mean by this? What is primary and secondary in this context?

Reply:
https://dev.mysql.com/doc/refman/5.7/en/group-replication-primary-secondary-replication.html#:~:text=There%20is%20a%20primary%20(source,in%20row%2Dbased%20replication).

Basically it's MySQL terminology for replicas.  If this is set to `true`, then a pvc will be created which persists the entire database between restarts unless you either manually delete it or clear your Kubernetes workloads.  For local development, this is generally not desirable-- so we avoid the behavior, much as we do for Kafka in local dev.